scripts/consensus.py

```diff
@@ -33,14 +38,171 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
         concat_df.columns.str.startswith("Metadata")
     ]
     concat_metadata_df = concat_df.loc[:, concat_metadata_cols]
+
     concat_df = concat_df.drop(concat_metadata_cols, axis="columns")
-    concat_df = pd.concat([concat_metadata_df, concat_df])
+    concat_df = pd.concat([concat_metadata_df, concat_df], axis="columns")
 
     # dropping columns with na values
     na_cols = get_na_columns(concat_df, cutoff=0)
     concat_df = concat_df.drop(na_cols, axis="columns")
 
-    return concat_df
+    # dropping Costes Features
+    costes_cols_to_drop = [x for x in concat_df.columns if "costes" in x.lower()]
+    concat_df = concat_df.drop(costes_cols_to_drop, axis="columns")
```

Comment:
We shouldn't hardcode this. (IIRC, costes features are only broken in one iteration of CP)

Reply:
I remember you have mentioned a function that deals with `costes` without the hard coding. I  completely forgot the name

Reply:
you can drop all hardcoded examples. They shouldn't be introduced into the `main` branch.